core/services/ocr2/plugins/ccip/internal/ccipdb/price_service.go

```diff
@@ -320,8 +322,10 @@ func (p *priceService) observeGasPriceUpdates(
 		return nil, fmt.Errorf("gasPriceEstimator is not set yet")
 	}
 
+	// Price getter should only return the source native token pricen as the tokens param is not empty
```

Comment:
```suggestion
	// Price getter should only return the source native token price as the tokens param is not empty
```

Do you mean that the `p.sourceNative` can not be empty?